src/bin/kafka_consumer.rs

```diff
@@ -15,6 +15,8 @@ struct Cli {
     processes: Option<usize>,
     #[arg(help = "Clear the queue of alerts already consumed from Kafka and pushed to Redis")]
     clear: Option<bool>,
+    #[arg(help = "Set a maximum number of alerts to hold in redis, default is 15000")]
+    max_in_queue: Option<usize>,
```

Comment:
I'm curious, did you run into an issue that inspired adding this, or is it just out of caution?

Reply:
Not really, just wanted to make it a parameter we can change, it was hardcoded before.